src/mono/mono/mini/interp/interp.c

```diff
@@ -3794,19 +3806,28 @@ mono_interp_exec_method (InterpFrame *frame, ThreadContext *context, FrameClause
 			memset (locals + ip [1], 0, ip [2]);
 			ip += 3;
 			MINT_IN_BREAK;
+#if HOST_BROWSER
```

Comment:
I think we should have this also on desktop. The problem is that on desktop we use computed goto so labels for all opcodes must be declared. A possible approach would be to start separating between IR level opcodes, like these, and normal interp opcodes. These IR level opcodes should be allocated after all normal opcodes so they don't waste space in the opcode table. We could define them separately with something like `OPDEF_IR`, or in a separate file, or something else if you have any ideas.